tests/src/blackBox/DockerActionTests.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package blackBox;
+
+import static common.WskCli.Item.Action;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.config.RestAssuredConfig;
+import com.jayway.restassured.specification.RequestSpecification;
+
+import common.BasicLauncher;
+import common.Pair;
+import common.TestUtils;
+import common.Util;
+import common.WhiskProperties;
+import common.WskCli;
+
+/**
+ * Basic tests of docker containers as actions.
+ */
+public class DockerActionTests {
+
+    public final static String helloPythonImage = "sjfink/hellopython";
+
+    @Rule
+    public TestRule watcher = TestUtils.makeTestWatcher();
+
+    private String makeContainerPayload(JsonElement payload) throws Exception {
+        String tmp = "{\"value\":" + payload.toString() + "}";
+        return new String(tmp.getBytes(), "UTF-8");
+    }
+
+    /**
+     * Run some tests on the hellopython container
+     */
+    @Test(timeout = 120 * 1000)
+    public void testHelloPython() throws Exception {
+        // kill and remove the container (if for some reason it's still there)
+        String containerName = "hellopython";
```

Comment:
@psuter your container test framework applies here, yes? better than this old test?